src/content/docs/formatter/editorconfig-support.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
I don't think there's a need for an entire page for editor config support. We could add a new section in the general guide